src/styles/blocks/_place.scss

```
@@ -0,0 +1,61 @@
+.place {
+  &__photos {
+    margin-bottom: 16px;
+  }
+  
+  &__photo-container {
+    position: relative;
+    
+    width: 100%;
+    max-height: 321px;
+    overflow: hidden;
+    
+
+    &:after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
+    }
+
+    @include onTablet {
+      height: 187px;
+      padding-bottom: 0;
+    }
+    
+    @include onDesktop {
+      height: 321px;
+      padding-bottom: 0;
+    }
+  }
+
+  &__photo {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+    transition: transform 1s;
```

use have var for this
```suggestion
    transition: transform $transition-duration;
```